app/presenters/publishing_api_presenters/link_extractor.rb

```diff
@@ -0,0 +1,84 @@
+module PublishingApiPresenters
+  class LinkExtractor
+    LINK_NAMES_TO_METHODS_MAP = {
+      document_collections: :document_collection_ids,
+      lead_organisations: :lead_organisation_ids,
+      organisations: :organisation_ids,
+      policy_areas: :policy_area_ids,
+      related_policies: :related_policy_ids,
+      statistical_data_set_documents: :statistical_data_set_ids,
+      supporting_organisations: :supporting_organisation_ids,
+      topics: :topic_content_ids,
+      world_locations: :world_location_ids,
+      worldwide_organisations: :worldwide_organisation_ids,
+      worldwide_priorities: :worldwide_priority_ids,
+    }
+
+    def initialize(item)
+      @item = item
+    end
+
+    def extract(filter_links)
+      filter_links.reduce(Hash.new) do |links, link_name|
+        method_name_on_item = LINK_NAMES_TO_METHODS_MAP[link_name]
```

Comment:
`method_name_on_item` is incorrect now (this used to go be used for `item.send()`).